wrap/tests/images/test_ImageContainerBySTLVector.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 import dgtal
 
+"""
```

Comment:
Why such string statement ?

Reply:
This the pythonic way of doing multiline comments. As mentionned in the PR, those test do not run -- and this is not because of GH actions.